test/sortByProps.js

```diff
@@ -0,0 +1,57 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('sortByProps', function() {
+  const alice = {
+    name: 'ALICE',
+    age: 101,
+    lastName: 'Smith',
+  };
+  const bob = {
+    name: 'Bob',
+    age: -10,
+    lastName: 'Smith',
+  };
+  const clara = {
+    name: 'clara',
+    age: 314.159,
+    lastName: 'Kris',
+  };
+  const people = [clara, bob, alice];
+
+  context('given an array of objects', function() {
+    specify(
+      'property to sort by exists, should return sorted values',
+      function() {
+        assert.deepEqual(RA.sortByProps(people)(['name']), [alice, bob, clara]);
+      }
+    );
+
+    specify(
+      'property to sort by does not exist, should return original list',
+      function() {
+        assert.deepEqual(RA.sortByProps(people)(['p']), people);
+      }
+    );
+
+    specify(
+      'sort by multiple properties, should return sorted values',
+      function() {
+        assert.deepEqual(RA.sortByProps(people)(['lastName', 'name']), [
+          clara,
+          alice,
+          bob,
+        ]);
+      }
+    );
+
+    specify('ignore properties that do not exist', function() {
+      assert.deepEqual(RA.sortByProps(people)(['lastName', 'p', 'name']), [
+        clara,
+        alice,
+        bob,
+      ]);
+    });
+  });
+});
```

Comment:
We're missing tests for currying. Example https://github.com/char0n/ramda-adjunct/blob/16b8831510a76c3f11c6480735710ad46d976983/test/divideNum.js#L14